source/common/protobuf/utility.h

```diff
@@ -263,9 +263,10 @@ class MessageUtil {
   static void loadFromJson(const std::string& json, ProtobufWkt::Struct& message);
   static void loadFromYaml(const std::string& yaml, Protobuf::Message& message,
                            ProtobufMessage::ValidationVisitor& validation_visitor);
+#endif
+
   static void loadFromFile(const std::string& path, Protobuf::Message& message,
```

Comment:
absolutely. If there end up being no other comments I'd prefer to do in a follow-up if OK?

something along the lines of 
"loadFromFile attempts to fill |message| from the file contents read from |path|
It will attempt to parse the file contents based on the file prefix, and the compiled in parsers, then defaulting to the last compiled in parser.. By default this will try in order binary proto, proto, yaml and json, though if yaml support is compiled out it will only try the first two.

Reply:
Doing this in a follow up is fine with me. I'd suggest being explicit about:

* `.pb` => binary proto
* `.pb_txt` => text proto (if supported)
* `.yml` or `.yaml` => YAML (if supported)
* otherwise JSON (if supported)